scripts/clippy_wasm.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# This scripts run clippy on the wasm32-unknown-unknown target,
+# using a special clippy_wasm.toml config file which forbids a few more things.
+
+set -eu
+script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$script_path/.."
+set -x
+
+mv clippy.toml clippy.toml.bak
+cp clippy_wasm.toml clippy.toml
+
+function cleanup()
+{
+    mv clippy.toml.bak clippy.toml
+}
+
+trap cleanup EXIT
```

Comment:
If we put `clippy_wasm.toml` in a directory: `clippy_wasm/clippy.toml`

It looks like we can just do this by setting an environment variable:
```suggestion
export CLIPPY_CONF_DIR="clippy_wasm"
```
See: https://github.com/rust-lang/rust-clippy/blob/ff843ac9aeb2fff9dbfae80d0801b16f4f676e5e/clippy_lints/src/utils/conf.rs#L475

Reply:
Ooooh niiiice!